### PR TITLE
Refactor fixtures from JSON to Python

### PR DESCRIPTION
**Pull request by SSPJ**
Does what it says on the tin. I've also added some new fixtures.

The motivation was that JSON fixtures are a bit fragile, seeing that they rely on hardcoded primary keys.

### CODE REVIEW

File: src/registrar/fixtures.py
Comment (SSPJ) [inline src/registrar/fixtures.py:44]: @michelle-rago @boonerang @calerubensteingsa @neilmb @h-m-f-t 

Currently refactoring (and expanding) our data fixtures and thought I'd ask again if any of you would care to have your account ID from the Login.gov sandbox included here.

Also, if you want some custom fixture data (beyond being an admin and having a few example domain applications), I'm happy to take requests. CC @loganmeetsworld  @igorkorenfeld on that last sentence.

Comment (michelle-rago) [reply src/registrar/fixtures.py:44]: @SSPJ What does adding our sandbox account here get us?

Comment (loganmeetsworld) [reply src/registrar/fixtures.py:44]: I'm good with what is there for me.

File: src/registrar/fixtures.py
Comment (loganmeetsworld) [inline src/registrar/fixtures.py:114]: These are a little funky when loaded for me - the fake data doesn't seem to be loaded when I go to the form.

**Review by loganmeetsworld — APPROVED** [review]
The Faker library is cool! I noted that I saw some things might've been missing. I wonder too if how the validation PR might work with this because of that but overall LGTM

File: src/registrar/fixtures.py
Comment (SSPJ) [reply src/registrar/fixtures.py:44]: @michelle-rago Unsure if you are asking what _this file_ does currently or what _data fixtures_ in general do, so answering both.

**Data fixtures:** We periodically have to recreate the database in staging or unstable. When we do that, we can run the code in this file to generate data. If you'd like to have fake data associated with your user, this is where we'd put it. An example might be a page that you can only access once you have a domain approved: if you'd like to check the wording on that page, you'd need to create an application, submit it, then approve it. A data fixture speeds the process by giving you an already approved domain.

**Currently:** The `DomainApplicationFixture` (further down in the file) randomly generates about 4 applications per user, in various states. If you have more specific requests, I'm happy to add them!

**Pull request merged**
